web/impact/impact/models/startup_role.py

```diff
@@ -15,11 +15,26 @@ class StartupRole(MCModel):
     FAST_TRACK = "Fast Track"
     AIR = "Alum In Residence"
     MC_STAFF = "MC Staff"
+    WINNER = "Winner"
+    GOLD_WINNER = "Gold Winner"
+    SILVER_WINNER = "Silver Winner"
+    PLATINUM_WINNER = "Platinum Winner"
+    DIAMOND_WINNER = "Diamond Winner"
+    INKIND_WINNER = "In-Kind Winner"
```

Comment:
Should match the constant defined in accelerate :)

Reply:
Hahahah!  Roger that!

Reply:
Done!